package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
```

Comment:
@ctong5 
@esme 

This description should contain text that describes what this repo/microservice does.